includes/class-module-base.php

```diff
@@ -3,6 +3,80 @@
 /**
  * Class WPLib_Module_Base
  */
-class WPLib_Module_Base extends WPLib {
+abstract class WPLib_Module_Base extends WPLib {
+
+	/**
+	 * Delegate calls to an instance class if the class has a INSTANCE_CLASS constant or plural name adds 's', otherwise delegate to WPLib.
+	 *
+	 * @param string $method_name
+	 * @param array $args
+	 *
+	 * @return mixed
+	 */
+	static function __callStatic( $method_name, $args ) {
```

Comment:
`$theme->the_link( $url, 'Link Text' );`  rather than `LawPress_Theme::the_link( $url, 'Link Text' );` when ever other reference in the file is `$theme`.  It will keep people from being confused why their code does not work when they expected it to.

It's just a convenience.